pkg/controllers/ippool/ippool_controller.go

```diff
@@ -134,6 +132,16 @@ func (r *IPPoolReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 			log.V(1).Info("added finalizer on ippool CR", "ippool", req.NamespacedName)
 		}
 
+		if obj.Spec.Type == "" {
```

Comment:
prefixLength should also be got from default subnet size?

Reply:
> prefixLength should also be got from default subnet size?
Actutally, PrefixLength is defined in SubnetRequest, it is specified by WCP.